Library/Formula/subversion.rb

```diff
@@ -70,6 +74,9 @@ class Subversion < Formula
   end
 
   def install
+    ENV.deparallelize
+    #Fixes: libtool:   error: error: relink 'libsvn_ra_serf-1.la' with the above command before installing it
+    #Please see issue https://github.com/Homebrew/linuxbrew/issues/408 for details
```

Comment:
Please add a space between the `#` and the comment. `brew audit --style subversion` should report this style issue.